packages/widgets/src/firewall/component.tsx

```diff
@@ -36,17 +37,11 @@ export default function FirewallWidget({ integrationIds, width }: WidgetComponen
   const initialSelectedFirewall = firewallsVersionData[0] ? firewallsVersionData[0].integration.id : "undefined";
   const isTiny = width < 256;
 
-  // State for managing the accordion
-  const [accordionValue, setAccordionValue] = useState<string | null>(() => {
-    const savedState = localStorage.getItem(`homarr-accordionState-${selectedFirewall || initialSelectedFirewall}`);
-    return savedState ? (JSON.parse(savedState) as string) : "interfaces";
+  const [accordionValue, setAccordionValue] = useLocalStorage<string | null>({
+    key: `homarr-${itemId}-firewall`,
+    defaultValue: "interfaces",
```

Comment:
Wouldn't it make more sense to just have `true` and `false` and `accordionOpened`

Reply:
I don't know if I'm not going to add another accordion later, so I choose to implement a solution allowing me to be able to handle multiple accordions and remember each one, from the beginning.